routing/missioncontrol.go

```diff
@@ -18,6 +19,9 @@ const (
 	// minSecondChanceInterval is the minimum time required between
 	// second-chance failures.
 	minSecondChanceInterval = time.Minute
+
+	// DefaultMaxMcHistory is the default maximum history size.
```

Comment:
Does this include failed attempts as well?

Reply:
In this PR we only store failed attempts.